packages/dialtone-vue3/localization/index.js

```diff
@@ -37,12 +37,17 @@ const localeManagerStorageKey = 'user-locale';
  * https://github.com/dialpad/goblin-client-tools/tree/main/packages/i18n#i18n-vue-3-compatible
  */
 export class DialtoneLocalization {
+  /**
+   * @prop {import('@dialpad/i18n').UseI18N} i18n
+   * @private
+   */
   constructor (locale = null) {
     if (typeof DialtoneLocalization.instance === 'object') {
       return DialtoneLocalization.instance;
     }
 
-    const app = getCurrentInstance().appContext.app;
+    // no good
+    //const app = getCurrentInstance().appContext.app;
```

Comment:
Yeah this vue3 internal. Happy to get rid of it. You can also remove the import `import { getCurrentInstance } from 'vue';`